.gitignore

```diff
@@ -1,6 +1,7 @@
 # General
 .vscode
 .idea
+.*.swp
```

Comment:
Other than the gitignore (just revert it), all seems fine to me.

Reply:
I rarely use git, so I'm not entirely sure I did the right thing.  I `git revert`'ed the commit hash and pushed the changes.  If some other action needs to be taken, let me know.